library/src/layout/page.rs

```diff
@@ -350,6 +359,16 @@ impl PageElem {
             // The padded width of the page's content without margins.
             let pw = frame.width();
 
+            // Make a clone of the original margin in case you need to modify
+            // the margins for a two-sided document.
+            let mut margin = margin.clone();
```

Comment:
Clippy seems to be complaining about this line as `margin` has a type that is `Copy`; does removing `.clone()` still make it work?
```suggestion
            let mut margin = margin;
```

Reply:
Yes. That solved it.

Reply:
Epic. (I can't resolve this thread for some reason, lol)

Please take a look at my other suggestion when you can, regarding spacing in the docs. ;)